gigareview/docker-compose.yml

```diff
@@ -2,10 +2,34 @@ version: '3.7'
 
 services:
 
+  web:
+    image: nginx:latest
```

Comment:
It's better to use a specific number insted of ``latest``, 
because of ``latest`` changes with each new releases, which creates two problems:
(1) a new version may introduce bugs
(2) our infrastructure is no longer immutable, as infrastructure elements may change independently of CI/CD pipeline deployment